index.html

```diff
@@ -1458,11 +1462,26 @@ <h2 id="OfflineAudioContext">
                   start rendering <code>length</code> sample-frames of audio
                   into <var>buffer</var>.
                   </li>
-                  <li>For every render quantum, check and suspend the rendering
-                  if necessary.
+                  <li>At the beginning of every render quantum, check scheduled
```

Comment:
So you can rework the prose by linking the right part of the processing algorithm, now, since it has merged.

Maybe beween 2. and 3. ?